spec/scrub/record_counter_spec.rb

```diff
@@ -15,14 +15,28 @@
   }
   let(:hundo) { 100 }
   let(:small_diff) { (Settings.scrub_line_count_diff_max * 100) - 1 }
-  def put_x_lines_in_file(x, file)
+
+  # Write a testfile with a certain number of { "ocn": i } ndjs.
+  def testfile(lines:, file:)
     File.open(file, "w") do |f|
-      1.upto(x) do |i|
-        f.puts(i)
+      1.upto(lines) do |i|
+        hash = {"ocn" => i}
+        f.puts hash.to_json
```

Comment:
I think I might compact this down to `f.puts({"ocn" => 1}.to_json)` or `f.puts({ocn: 1}.to_json)` (I was going to suggest a version without parentheses not realizing `puts` would get confused by the hash literal). Line 34 as well.